Dockerfile

```diff
@@ -0,0 +1,140 @@
+# All build arguments
+
+# TARGETPLATFORM - automatic assignment via `docker build --platform xyz`
+# RUST_PROFILE - defaults to release
+
+# Build babylon-node binary
+FROM debian:11-slim AS binary-build-stage
+LABEL org.opencontainers.image.authors="devops@radixdlt.com"
+LABEL org.opencontainers.image.description="Java + Debian 11 (OpenJDK)"
+
+ENV DEBIAN_FRONTEND noninteractive
+
+CMD ["/bin/bash"]
+
+RUN  \
+  apt-get update && apt-get install -y --no-install-recommends \
+  docker.io=20.10.5+dfsg1-1+deb11u2 \
+  libssl-dev=1.1.1n-0+deb11u4 \
+  pkg-config=0.29.2-1 \
+  unzip=6.0-26+deb11u1 \
+  wget=1.21-1+deb11u1 \
+  software-properties-common=0.96.20.2-2.1 
+RUN add-apt-repository -y ppa:openjdk-r/ppa && \
+  apt-get install -y --no-install-recommends \
+  openjdk-17-jdk=17.0.6+10-1~deb11u1
+
+#install Gradle
```

Comment:
Overly nit-picky comment - but it might be nice if all our comments in this file follow the same pattern - eg `# Hello` - hash, space, capital first letter?